taxi/views.py

```diff
@@ -36,6 +36,17 @@ class ManufacturerListView(LoginRequiredMixin, generic.ListView):
     template_name = "taxi/manufacturer_list.html"
     paginate_by = 5
```

Comment:
The number of items to paginate by is hardcoded here. It would be better to move this to a configuration setting, so that it can be easily changed if needed.